docs/extending-locust.rst

```diff
@@ -28,13 +28,43 @@ Event listeners can be registered at the module level in a locust file. Here's a
 Adding Web Routes
 ==================
 
-Locust uses Flask to serve the web UI and therefore it is easy to add web end-points to the web UI. 
+Locust uses Flask to serve the web UI and therefore it is easy to add web end-points to the web UI.
 Just import the Flask app in your locustfile and set up a new route::
 
     from locust import web
-    
+
     @web.app.route("/added_page")
     def my_added_page():
         return "Another page"
 
 You should now be able to start locust and browse to http://127.0.0.1:8089/added_page
+
+
+Enabling SSL
+==================
+
+The Locust web interface also supports SSL encrypted connections. This is especially useful
+if, for instance, you're testing an API which may incidentally expose personal information
+or application secrets through the web interface.
```

Comment:
I still question the value for the wider locust audience. SSL encrypted UI may prevent MITM, but it's irrelevant if the page is unsecured. Related: https://github.com/locustio/locust/pull/284. My concern here is the same as the comment there. In my opinion, it's cleaner and more secure to attach layers of security on top of locust rather than trying to make the locust page secure. If the encryption and auth are important for your organization, the time investment to set up nginx + let's encrypt with basic auth is relatively small.